Bottom: 730bbff64460aef7d9aab864ef7677de8dc573cc
Top:    9d2dfce0840ad54e812f72135522ebf9cecfa52b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:54:58 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
